app/src/main/java/com/duckduckgo/app/tabs/db/TabsDao.kt

```diff
@@ -35,12 +36,15 @@ abstract class TabsDao {
     @Query("select * from tabs inner join tab_selection on tabs.tabId = tab_selection.tabId order by position limit 1")
     abstract fun liveSelectedTab(): LiveData<TabEntity>
 
-    @Query("select * from tabs order by position")
+    @Query("select * from tabs where deletable is 0 order by position")
     abstract fun tabs(): List<TabEntity>
 
-    @Query("select * from tabs order by position")
+    @Query("select * from tabs where deletable is 0 order by position")
     abstract fun liveTabs(): LiveData<List<TabEntity>>
 
+    @Query("select * from tabs where deletable is 1 order by position")
+    abstract fun deletableLiveTabs(): Observable<List<TabEntity>>
```

Comment:
The problem with `LiveData` is that it will behave as a `BehaviorSubject` returning always the last value. Which is not what I want.
I guess I can move to `Flow`. Will give it a go

Reply:
Yep, just suggesting to use `LiveData` to save you some time exploring `Flow`.  Check the usage of `ViewStateConditionalRender.renderIfChanged` in case it can solve the problem of receiving twice the same value.

Reply:
Yeah, saw that. I personally dislike `renderIfChanged`. I saw it and thought is making things more complex, keeping and tracking state in a class, just because we want to use `LiveData` in a place that we prob should not.
I think `Flow` should be able to resemble that `Observable`, will go that way instead